gen3-integration-tests/tests/test_dataguids.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pytest
+import requests
+
+from cdislogging import get_logger
+from playwright.sync_api import expect
+import utils.gen3_admin_tasks as gat
+
+logger = get_logger(__name__, log_level=os.getenv("LOG_LEVEL", "info"))
+
+# TODO : enable this test after the manifest PRs are ready to roll
+@pytest.mark.wip
+class TestDataGuids:
+    variables = {}
+    dataguids_url = "https://dataguids.org"
+    non_existent_guids = [
+        "dg.ABCD/0000b4b4-2af4-42e2-9bfa-6fd11e5fb97a",
+        "0000b456-3r56-1dr3-0rt4-6fd11e5fb97a",
+    ]
+
+    @classmethod
+    def setup_class(cls):
+        cls.variables["guids"] = []
+        logger.info("Getting all the hosts from manifest.json ...")
+        hosts_list = gat.get_dataguids_hosts
```

Comment:
don't we already run this and save manifest.json in test data folder? can't we use that instead of creating a new job?